service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ConditionLinkSetMapper.java

```diff
@@ -72,6 +74,15 @@ public String mapConditionToLinkSet(Condition condition, boolean isNested) {
 
         builder.code(buildCode(condition));
 
+        if (condition.hasAsserter()) {
```

Comment:
I don't see any tests for a condition without an asserter? Otherwise, if this is mandatory, maybe we should throw an exception when there's no asserter?

Reply:
I also agree, believe asserter is a mandatory field (from the GP connect API https://gpc-structured-1-5-0.netlify.app/accessrecord_structured_problems.html) thus should throw an exception when asserter is not present

Reply:
Similarly, only Practitioner is profiled for the reference. Since we check the reference type in line 80 we should also throw and exception if its not a practitioner instead of silently skipping the field.